config/environments/development.rb

```diff
@@ -17,4 +17,8 @@
   config.eager_load = false
 
   config.whiny_nils = true
+
+  config.hosts += [
```

Comment:
Not sure if this is similar to the draft app thing and it needs to be added?
https://github.com/alphagov/govuk-docker/blob/master/projects/feedback/docker-compose.yml#L36